src/customize-pop/customize-desktop.md

```diff
@@ -0,0 +1,99 @@
+# Customizing the Desktop
+
+Change the function of the `SUPER` key, desktop backgrounds, workspace features, and placement and visibility of desktop components in `Settings` ➞ `Desktop`.
+
+- [`Desktop Options`](/customize-pop/customize-desktop.md#desktop-options) provides options to modify the `SUPER` key's function, enabling Hot Corner, and adding or positioning top bar elements.
+- [`Background` settings](/customize-pop/customize-desktop.md#background) allows users to set pre-installed wallpaper or user images as the background image.
+- [`Appearance` settings](/customize-pop/customize-desktop.md#appearance) provides the option to choose a dark or light theme that applies across the entire desktop environment.
+- [`Dock` settings](/customize-pop/customize-desktop.md#dock) provides option to modify Dock visibility, placement, size, and displayed icons.
+- [`Workspaces` settings](/customize-pop/customize-desktop.md#workspaces) includes options configure workspace positioning and behavior.
+
+---
+
+## Desktop Options
+
+Customize basic desktop elements in the `Desktop Options` tab.
+
+![General Settings](/images/customize-desktop/general-settings.png)
+
+| Setting | Function |
+|----------|----------|
+| Super Key Action | Assigns the `SUPER` key to start the Launcher, display Workspace navigation, or view installed applications. |
+| Hot Corner | Displays workspace navigation when the user moves the mouse cursor to the upper left corner of the screen. |
+| Top Bar | Display or hide items along the top bar of the main home screen. |
+| Window Controls | Enable or disable the minimize and maximize buttons in system and application windows. |
```

Comment:
`system and application windows` may be more verbose than it needs to be. The option applies to all windows (except for those using CSD), there's no way to set it separately for "system windows" and "application windows" even if there was a distinction between those things.

Reply:
I got into the habit of distinguishing the two from a previous comment, I forget the context. Whenever I write "...in windows" I'm worried someone might think it refers to the OS, Microsoft Windows, since referring to  MS Windows simply as "Windows" is common. If I clarify by writing "application windows" it excludes windows that are not spawned by applications. Maybe I'm overthinking it.

Reply:
Yeah, we can't avoid the word "windows" just because Microsoft picked it as the name of their OS, imo. Either just say "application windows" since there's no clear distinction between "applications" / "system applications" / things that aren't those anyway, or just say "windows." It's not capitalized, there is no reason someone should think you're naming the OS, especially in this context.

In this case, you could say `in window titlebars` since that is where the buttons we're referring to are.